React/src/interfaces/RegisterUser.ts

```diff
@@ -0,0 +1,8 @@
+export interface RegisterUser {
+  email: string;
+  password: string;
+  passwordConfirmation: string;
+  name: string;
+  lastName: string;
+  locale: 'string';
```

Comment:
Based on:
```
lastName: string;
locale: 'string';
```

Keep in mind that the type `string` is not the same as `"string"`.
Every type wrapped in "" means that it accepts that string literal (that word specifically) as it's type.
It is useful when you need to pass only some values to a props, like for example:
`color: "red" | "blue" | "green"` so if we pass "orange" to this prop TS will throw a type error

```suggestion
  locale: string;
```